omrmakefiles/rules.linux.mk

```diff
@@ -57,7 +57,11 @@ endif
 
 # Compile without exceptions
 ifeq (gcc,$(OMR_TOOLCHAIN))
-    GLOBAL_CXXFLAGS+=-fno-exceptions -fno-rtti -fno-threadsafe-statics
+ifeq (1,$(OMR_RTTI))
+    GLOBAL_CXXFLAGS+=-fno-exceptions -fno-threadsafe-statics
```

Comment:
There appears to be inconsistent spacing/tabbing between line 61 and 63.  Would you be able to update this?